<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericMakeStep</class>
 <widget class="QWidget" name="GenericMakeStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>392</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="makeLabel">
     <property name="text">
      <string>Override %1:</string>
     </property>
     <property name="buddy">
      <cstring>makeLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="makeLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="makeArgumentsLabel">
     <property name="text">
      <string>Make arguments:</string>
     </property>
     <property name="buddy">
      <cstring>makeArgumentsLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="makeArgumentsLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="targetsLabel">
     <property name="text">
      <string>Targets:</string>
     </property>
     <property name="buddy">
      <cstring>targetsList</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="targetsList"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
